com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -156,6 +164,47 @@ public static void DrawMultipleFields(GUIContent label, SerializedProperty[] ppt
             EditorGUIUtility.labelWidth = labelWidth;
         }
 
+        /// <summary>
+        /// Draw a multiple field property
+        /// </summary>
+        /// <param name="label">Label of the whole</param>
+        /// <param name="lbls">The labels mapping the values</param>
+        /// <param name="values">The values to be displayed</param>
+        public static void DrawMultipleFields<T>(GUIContent label, GUIContent[] lbls, T[] values)
```

Comment:
same comment as above between from 'lbls' to 'labelList' or similar ?

Reply:
changed, thank you